war/pom.xml

```diff
@@ -305,13 +305,6 @@ THE SOFTWARE.
                   <version>3.1.5</version>
                   <type>hpi</type>
                 </artifactItem>
-                <artifactItem>
-                  <!-- detached after 1.547 -->
-                  <groupId>org.jenkins-ci.plugins</groupId>
-                  <artifactId>windows-slaves</artifactId>
-                  <version>1.8.1</version>
-                  <type>hpi</type>
-                </artifactItem>
```

Comment:
It is helpful to run `LoadDetachedPluginsTest#noUpdateSiteWarnings` when updating this list, but unfortunately that test has to be `@Ignore`d by default to avoid failing unrelated builds when new security warnings are introduced. As a courtesy I ran that test for you, but it would be helpful to keep this in mind for future changes.

Reply:
Thanks.  As practice for myself, I removed the `@Ignore` on that test method and ran it to confirm that it passes.